File: Swift/Lista_04_EX04/Usuario.swift
import Foundation
 
 public class Usuario{
 
    public var login: String
     
    init(login: String) {
        self.login = login
    }
  
	get pegaLogin()
	{
		return login
	}
	
	func descricao( ) {
		print("Base para os outros Usuarios")
	}
	
	func boasVindas( ) {
		print("Bem-vindo: \ (login)")
	}
	
}

File: Java/Lista 4/L4_Ex1.4/src/l4_ex1/pkg4/Usuario.java

package l4_ex1.pkg4;

public class Usuario {
    private String login;

    public Usuario(String login) {
        this.login = login;
    }

    public String getLogin() {
        return login;
    }
    
    public void descricao(){
        System.out.println("Base para os outros Usuarios");
    }
    
    public void boasvindas(){
        System.out.println("Bem-vindo " + this.login );
    }
     
}
